### PR TITLE
[Polaris] Update replicas for polaris dashboard and webhook deployment

### PR DESCRIPTION
**Pull request by makoscafee**
**Why This PR?**
_a short description of why this PR is needed_
This PR update dashboard and webhook deployment replicas to 2 to match new added rule enabled by default in polaris `deploymentMissingReplicas`

Fixes #
- Fixes `deploymentMissingReplicas` check error 
**Changes**
Changes proposed in this pull request:

* Update replicas for webhook and dashboard to `2`
*

**Checklist:**

* [ ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/polaris/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.3
-appVersion: "4.2"
+version: 5.0.0
+appVersion: "5.0"
```
Comment (sudermanjr) [inline stable/polaris/Chart.yaml:5]: I don't think the version of polaris is changing, so we should keep this the same I think

File: stable/polaris/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.3
-appVersion: "4.2"
+version: 5.0.0
```
Comment (sudermanjr) [inline stable/polaris/Chart.yaml:4]: This doesn't seem like a major change to the chart. Honestly it feels more like a patch version to me

**Review by sudermanjr — CHANGES_REQUESTED** [review]

**Review by rbren — APPROVED** [review]

**Comment by rbren** [comment]
Should we try updating kube-hunter to get rid of that vuln?

**Comment by makoscafee** [comment]
> Should we try updating kube-hunter to get rid of that vuln?

Sure, I can do that in this PR.

**Pull request merged**